src/api/app/models/notification.rb

```diff
@@ -3,6 +3,11 @@ class Notification < ApplicationRecord
   MAX_RSS_ITEMS_PER_GROUP = 10
   MAX_PER_PAGE = 300
 
+  EVENT_TYPES = ['Event::CreateReport', 'Event::ReportForRequest', 'Event::ReportForProject', 'Event::ReportForPackage', 'Event::ReportForComment',
+                 'Event::ReportForUser', 'Event::ClearedDecision', 'Event::FavoredDecision', 'Event::AppealCreated'].freeze
+
```

Comment:
Using `to_s` on the class name in theses cases rather than magic strings really helps refactoring tools make sense of it, like this:

```
EVENT_TYPES = [Event::CreateReport.to_s, Event::ReportForRequest.to_s, ...].freeze
```

Reply:
What about using `%w()` ?

Reply:
I usually use RubyMine, but basically anything built on RubyParser will fail to identify constants embedded in strings.

I don't think %w parses as constants with RubyParser, but I like that much better.

This is not something I feel strongly about, it's just a habit I've picked up because automated refactoring is core to my own workflow.

Reply:
Alright, I like constants too, so I think I'll just change it